### PR TITLE
Feature/favourites follows translation additions

### PR DESCRIPTION
**Pull request by hammol04**
Hey guys,

Please can I get a sanity check for these translations (English, Welsh, Scots Gaelic and Irish Gaelic).

Alphabetise script has been run.

@BjornTwachtmann @djmarland @alexgisby @jijo78 @pris54 @sara89sgm 

Thanks
Laurence

### CODE REVIEW

File: src/RMP/Translate/lang/programmes/en.po [hunk 1]
```diff
@@ -1702,6 +1727,16 @@ msgstr "Show less"
 msgid "show_more"
 msgstr "Show more"
 
+#. English: "Show More Added Episodes and Clips".
+#. Priority for: Favourites
+msgid "show_more_added_episodes_and_clips"
+msgstr "Show More Added Episodes and Clips"
```
Comment (djmarland) [inline src/RMP/Translate/lang/programmes/en.po]: Should be sentence case: "Show more added episodes and clips"
If you look around, you'll see all the other translations are too.

**Comment by hammol04** [comment]
Updated, cheers @djmarland

File: src/RMP/Translate/lang/programmes/en.po [hunk 2]
```diff
@@ -495,6 +500,11 @@ msgstr[0] "Episodes"
 msgstr[1] "Episode"
 msgstr[2] "Episodes"
 
+#. English: "Episodes and Clips".
+#. Priority for: Favourites
+msgid "episodes_and_clips"
+msgstr "Episodes and Clips"
```
Comment (djmarland) [inline src/RMP/Translate/lang/programmes/en.po]: Here's another

File: src/RMP/Translate/lang/programmes/en.po [hunk 3]
```diff
@@ -1860,6 +1900,12 @@ msgstr "View earlier today"
 msgid "view_episodes"
 msgstr "View episodes"
 
+#. English: "%1 Added".
+#. %1 is a variable, which may contain the value: "3".
+#. Priority for: Favourites
+msgid "x_added"
+msgstr "%1 Added"
```
Comment (djmarland) [inline src/RMP/Translate/lang/programmes/en.po]: and another ;-)

**Comment by hammol04** [comment]
Updated. Thanks

**Comment by jijo78** [comment]
:shipit:

**Pull request merged**